.travis.yml

```diff
@@ -25,20 +27,27 @@ before_install:
 
   # set UID to run docker service with
   - echo "UID=$(id --user)" > .env
-
-install:
-  - docker-compose up -d --build
-  - npm install @commitlint/{config-conventional,travis-cli}
-
-script:
-  - docker-compose exec caluma black --check .
-  - docker-compose exec caluma flake8
-  - ./node_modules/.bin/commitlint-travis
-  - docker-compose exec caluma ./manage.py makemigrations --check --dry-run --no-input
-  - docker-compose exec caluma pytest --no-cov-on-fail --cov --create-db -vv
-  - docker-compose exec caluma reuse lint
-after_success:
-  - git config --global user.name "semantic-release (via TravisCI)"
-  - git config --global user.email "semantic-release@travis"
-  - pip install python-semantic-release
-  - semantic-release publish
+jobs:
+  include:
+    - env: BUILD_TYPE=standard
+      install:
+        - docker-compose up -d --build
+        - npm install @commitlint/{config-conventional,travis-cli}
+
+      script:
+        - docker-compose exec caluma black --check .
+        - docker-compose exec caluma flake8
+        - ./node_modules/.bin/commitlint-travis
+        - docker-compose exec caluma ./manage.py makemigrations --check --dry-run --no-input
+        - docker-compose exec caluma pytest --no-cov-on-fail --cov --create-db -vv
+        - docker-compose exec caluma reuse lint
+      after_success:
+        - git config --global user.name "semantic-release (via TravisCI)"
+        - git config --global user.email "semantic-release@travis"
+        - pip install python-semantic-release
+        - semantic-release publish
+    - env:
+        BUILD_TYPE=pipenv
+        PIPENV_IGNORE_VIRTUALENVS=1
+      install: python3 setup.py pipenv
+      script: pipenv run pytest
```

Comment:
Not entirely sure what that does.
Why would we need another CI job for running pytest?
This is already done in the first job using docker-compose, no?

Reply:
* If the python version is not set right, the test might fail. 
* If the code in setup.py somehow breaks
** because pipenv had changed its interface, the test might fail
** because distribute or setuptools changed its interface, the test might fail

There are a lot of ways the changes in this PR might fail. If we test it, we can fix it. If we don't test it, a user or contributor might have an issue and he won't know if he has done something wrong or if it is broken. We don't want to deliver broken things to users!